### DISCUSSION TITLE
ABS Keeps stopping in docker on unraid

### DISCUSSION THREAD

**Discussion by brianalsum** [advplyr/audiobookshelf#2715] [Q&A]
I keep having the ABS docker stop every night on my unraid server. I've removed it, deleted the install image and reinstalled it, and reinstalled but so far nothing has helped. I've looked through the logs and can't find anything that would cause it to stop. I want to delete all the app data but I'm worried that I will lose all my library info. is there a way to save that info (Series, Book Art, Titles... etc)

**Comment by nichwall** [comment]
Have you set up Docker mounts? If you have them your stuff is separate from the container.

What server version? Have you enabled the debug logs?
